scipy/stats/meson.build

```diff
@@ -14,35 +14,17 @@ stats_special_cython_gen = generator(cython,
     _stats_pxd,
     cython_special_pxd])
 
-statlib_lib = static_library('statlib_lib',
-  [
-    'statlib/ansari.f',
-    'statlib/spearman.f',
-    'statlib/swilk.f'
-  ],
-  fortran_args: _fflag_Wno_unused_variable
-)
-
-statlib_module = custom_target('statlib_module',
-  output: ['_statlib-f2pywrappers.f', '_statlibmodule.c'],
-  input: 'statlib.pyf',
-  command: [py3, generate_f2pymod, '@INPUT@', '-o', '@OUTDIR@']
-)
-
-statlib = py3.extension_module('_statlib',
-  statlib_module,
-  c_args: numpy_nodepr_api,
-  fortran_args: fortran_ignore_warnings,
-  link_with: statlib_lib,
-  dependencies: [fortranobject_dep],
+_stats = py3.extension_module('_stats',
```

Comment:
Not sure why GH is showing the diff as such but I basically removed only the statlib parts.